### ISSUE TITLE
No "Nautilus integration" option? (But it still works.)

### ISSUE THREAD

**Issue by thoughton** [andyholmes/gnome-shell-extension-gsconnect#584]
Hi,

I'm a new user.

Whilst reading the documentation (and other issues) I keep seeing mentions of an "Enable Nautilus integration" option, but I cannot seem to find it anywhere...

However my Nautilus integration _does_ work (I can "Send to Mobile Device" via the right-click menu), so I was just wondering really whether the documentation might be out of date?

Or - if the option does still exist - I was hoping someone could perhaps just finally help me find it! :)

Thanks for a great extension.

**Comment by ferdnyc** [comment]
@thoughton Yeah, you're right — that part of the docs could use updating. In the current code the Nautilus hooks (and soon Nemo, as well) are auto-installed when the extension starts up, and they'll automatically be used as long as the `nautilus-python` / `nemo-python` plugin is installed.

**Comment by thoughton** [comment]
Ace, thanks a lot for confirming!

I guess a trickier thing is what to do (if anything) about the many other Issues on here that also mention the "Nautilus integration" in their replies? I had a problem with the integration to begin with, which in the end turned out to be caused by an issue with my local python installation, but before I realised that I spent quite a lot of time reading old Issues on here and becoming increasingly confused by the mentions of a "Nautilus integration" option which I could not find...

However perhaps if the docs were updated to mention that there used to be such an option, but it has now been removed, that would go a good way towards resolving the situation.

Thanks again.
